Plugin/CppApi/source/CoordinateConversion/CoordinateConversionController.cpp

```diff
@@ -399,9 +402,11 @@ void CoordinateConversionController::setCaptureMode(bool captureMode)
 }
 
 /*!
-  \brief Handles the mouse click at \a mouseEvent .
+  \fn void CoordinateConversionController::onMouseClicked(QMouseEvent& mouseEvent);
+  \brief Signal emitted when the mouse is clicked.
 
-  If the tool is active and in \l captureMode, this will be used as the input for conversions.
+  If the tool is active and is in \l captureMode, the clicked location will be used
+  as the input for conversions.
```

Comment:
`onMouseClicked` is actually a slot.  So it should more be handled like a regular method that gets called when the view's `mouseClicked` signal occurs.  Something like that.